### ISSUE TITLE
PAW: Results screens build & logic

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#15288]
### User story
AS A Veteran using the PACT Act wizard
I WANT to receive the appropriate results screen for my answers
SO THAT I know if I may be eligible for benefits

### Engineering notes / background 
- Per [this conversation](https://kindsys.slack.com/archives/C03EUQRCHGS/p1695414248735699?thread_ts=1695413541.226259&cid=C03EUQRCHGS), Results screen 1's content will remain as-is for now; separate ticket to be created for dynamic accordions and final content
- Need to duplicate Results screen 1 and put on a different route (`results-1-1-b`) 
  - This way, we can plug in Results screen logic without disrupting Cindy's research
- Build Results screens 2 and 3 with dummy content
- Implement the display logic for all 3 flows (1989 or earlier, 1990 or later, during both of these periods).

### Testing requirements
- Add unit tests for all added functionality
- Add Cypress tests for all added functionality

### Design
[Sketch file](https://sketch.com/s/2bf4405b-5cc4-4672-88fb-d215abdf3b0d)

## Acceptance criteria
- [x] Results screens 2 and 3 are built (using placeholder copy that clearly states which results screen it is)
- [x] Logic implemented for all 3 Results screens according to designs and logic flow
- [x] Unit tests added
- [x] Designer reviewed
- [x] PM reviewed

**Comment by wesrowe** [comment]
@randimays, as we didn't get to this in refinement today (sorry I didn't see the slack traffic till now), can you please pre-refine this one?

**Comment by wesrowe** [comment]
@thejordanwood, Jill raised a question in the details above: 
> Should there be a results screen 3 in the Sketch file?

I think the answer is "yes"...

**Comment by thejordanwood** [comment]
@wesrowe I've put results screens 2 and 3 in [the Sketch file](https://sketch.com/s/2bf4405b-5cc4-4672-88fb-d215abdf3b0d). 

Note: I originally had confusing labels in my Sketch file and was calling the Results 1.1 and 1.2 screens "Results 1 and 2" because I was using the labels to show the screen order in the prototype. I've corrected the labels and now all the results screen should be in there.

**Comment by randimays** [comment]
I am getting close to finishing. Since this is the last large chunk of functionality to add to the PACT Act wizard, I'm doing some thorough testing, both Cypress and unit as well as manual testing. I've discovered an issue I'll need some help in resolving: https://dsva.slack.com/archives/C52CL1PKQ/p1696887224281889. Hoping to have an answer for that before lunch tomorrow.

**Comment by randimays** [comment]
Just posted for review ([Slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1696972874916179)). This one is intended to wrap up MVP (with placeholder content) functionality, so we'll likely need a day or two to get all the testing and approvals in.

**Comment by jilladams** [comment]
@randimays (and @cindymerrill ) FYI that IA gave us permission to move forward with Proposed path = va.gov/pact-act-eligibility for our URL. 

Randi, I know you had been holding off on URL changes for the sake of Cindy's review screen access while doing synthesis. Just let me know if you'd like a separate ticket to get that updated -- I am commenting that here, bc if we can move ahead, you might be able to tuck it into your PR for this ticket.

**Comment by randimays** [comment]
@jilladams That's going to involve some changes across content-build, devops and vets-website. Let's ticket separately please

**Comment by jilladams** [comment]
Done! https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15669 Can refine it next week.

**Comment by cindymerrill** [comment]
Is there a new URL for the R1 results page that I've been using in research? If so, I could switch the prototype to point to that one, if that would unblock @randimays 's implementation. Also, I took screenshots of the R1 results pages, so I technically don't need the page live on Staging anymore.

**Comment by randimays** [comment]
Validated in staging today. Closing!

@cindymerrill If you still need access to the prototype for the time being, you should keep your current URL. The Results 1 (pages 1 and 2) that were built in _this_ ticket do not allow a deep dive and have only placeholder content.
